labs/wk06/lab_06_source.R

```diff
@@ -0,0 +1,787 @@
+                     dep.var.labels = "Log Median Home Value",
+                     type = S_TYPE,
+                     digits = 2)
+
+
+treat2 <- as.numeric( d.fed$num.lihtc > 0 )
+
+# add DV and IVs into one data frame per time period
+dl1 <- data.frame( y=y1, treat=treat2, post=0, p.col=p.col00, p.unemp = p.unemp00, p.black = p.black00)
+dl2 <- data.frame( y=y2, treat=treat2, post=1, p.col=p.col10, p.unemp = p.unemp10, p.black = p.black10)
+
+# stack the time periods together into one data frame
+dl3 <- rbind( dl1, dl2 )
+data_new_l <- dl3                                   # Duplicate data
+data_new_l[is.na(data_new_l) | data_new_l == "Inf" | data_new_l == "-Inf"] <- NA  # Replace NaN & Inf with NA
+
+
+
+m <- lm( y ~ p.col +p.unemp +p.black + treat + post + treat*post, data=data_new_l)
+
+###labels for knitting
+PercentCollegeGrad <- data_new_l$p.col
+PercentUnemployed <- data_new_l$p.unemp
+PercentBlackPopulation <- data_new_l$p.black
+
+# view regression results
+pander(summary(lm( y ~ PercentCollegeGrad +PercentUnemployed + PercentBlackPopulation + treat + post + treat*post, data=data_new_l)))
+
+# display model results
+stargazer::stargazer(m,
+                     title ="LIHTC Treatment",
+                     covariate.labels = c("Percent College Grads", "Percent Unemployed", "Percent Black Population"),
+                     dep.var.labels = "Log Median Home Value",
+                     type = S_TYPE,
+                     digits = 2)
+
+# create a variable that identifies if a tract received NMTC funding
+treat_t <- as.numeric( d.fed$num.nmtc > 0 )
+
+# store the year 2000 data
+dt1 <- data.frame( y=y1, treat=treat_t, post=0 )
+# store the year 2010 data
+dt2 <- data.frame( y=y2, treat=treat_t, post=1 )
+
+# stack the two time periods together
+dt3 <- rbind( dt1, dt2 )
+
+# create a variable that identifies if a tract received NMTC funding
+treat_t1 <- as.numeric( d.fed$num.lihtc > 0 )
+
+# store the year 2000 data
+dt4 <- data.frame( y=y1, treat=treat_t1, post=0 )
+# store the year 2010 data
+dt5 <- data.frame( y=y2, treat=treat_t1, post=1 )
+
+# stack the two time periods together
+dt6 <- rbind( dt1, dt2 )
+dt7 <- rbind( dt4, dt5 )
+
+m <- lm( y ~ treat + post + treat*post, data=dt6)
+m2 <- lm( y ~ treat + post + treat*post, data=dt7)
+
+
+#### NMTC Regression Summary 
+# view regression results
+pander(summary( m ))
+
+#### LIHTC Regression Summary
+pander(summary(m2))
+
+# display model results
+stargazer::stargazer(m,m2,
+                     title  = "NMTC and LITHC Comparison",
+                     dep.var.labels = "Log Median Home Value",
+                     column.labels = c("NMTC", "LIHTC"),
+                     type = S_TYPE,
+                     digits = 2)
+
+### Home value test
+
+home1 <- 200000
+home2 <- 100000
+
+rate.of.return <- .1/(1:10)
+
+roi.1 <- home1*rate.of.return
+roi.2 <- home2*rate.of.return 
+
+plot( 1:10, roi.1, type="b", pch=19, bty="n",
```

Comment:
We will also need to adjust some codes within the plot so that we can make the trend lines visible